src/exportImage.py

```diff
@@ -121,49 +147,92 @@ def delete_local_directory(self, directoryName):
         self.log.info(
             f'Folder "{directoryName}" in path: "{directory_path}" removed successfully')
 
-    def create_geopackage(self, bbox, filename, url, taskid, fullPath, resolution):
+    def create_geopackage(self, bbox, filename, url, task_id, full_path, max_zoom, resolution):
+        warp_result = self.create_geopackage_base(
+            bbox, filename, url, task_id, full_path, resolution)
+
+        if not warp_result:
+            self.log.error(f'gdal return empty response for task: "{task_id}"')
+            return False
+
+        overviews_result = self.create_geopackage_overview(
+            filename, task_id, full_path, max_zoom)
+
+        # Clear cache (for closing process)
+        warp_result.FlushCache()
+        warp_result = None
+
+        # Check for overview build errors
+        if overviews_result == GDALBuildOverviewsResponse.CE_Failure or overviews_result == GDALBuildOverviewsResponse.CE_Fatal:
+            self.log.error(
+                f'gdal return empty response for task: "{task_id}" (overviews)')
```

Comment:
its not empty, why not log the "overviews_result "?

Reply:
Added